cli/module_registry.go

```diff
@@ -137,8 +136,10 @@ func UpdateModuleAction(c *cli.Context) error {
 		return err
 	}
 
-	moduleID, err := updateManifestModuleIDWithArgs(c, client, manifest.Name, publicNamespaceArg, orgIDArg)
```

Comment:
okay so i removed this function because I wasn't really sure if this was something we need anymore. My understanding is that the moduleID is created on the APP B.E. in APP on create, and so if its not set in the Manifest file (which its not but should be after this PR) we should grab it based on OrgId / public namespace / and module name from the BE so we don't just guess it every time. 

Am I missing something here?

Reply:
The logic for this function can be simplified but I think it is still required to some degree (for upload)